src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3875,16 +3875,16 @@
      *
      * <p>parses as:</p>
      * <pre>
-     * +--LITERAL_NEW (new)
-     *     |
-     *     +--IDENT (ArrayList)
-     *     +--LPAREN (()
-     *     +--ELIST
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (50)
-     *     +--RPAREN ())
+     * LITERAL_NEW -&gt; new
```

Comment:
why this changes are here? they are related to `LITERAL_NEW`

Reply:
I did pull request of LITERAL_NEW yesterday and I don't know why it is showing here too.  I am trying to resolve this.